aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -700,16 +684,41 @@ module aptos_token::token {
         }
     }
 
+    public fun create_royalty(royalty_points_numerator: u64, royalty_points_denominator: u64, payee_address: address): Royalty {
+        assert!(royalty_points_denominator != 0, error::invalid_argument(EINVALID_ROYALTY_NUMERATOR_DENOMINATOR));
```

Comment:
nit: royalty_points_denominator > 0

Reply:
bc u64 is unsigned I guess it'll always >=0 ?